docs/NEW_LANGUAGE_FEATURE_INTEGRATION_PROCESS.md

```diff
@@ -36,18 +40,18 @@ on existing checks.
 
```

Comment:
Not for this line, but similar to above: we should have a list of general concepts here to guide the reader. Example: check acceptable tokens...

Reply:
done